draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -10018,6 +10021,24 @@ Content-Encoding: gzip
   <seriesInfo name="ACM Transactions on Internet Technology" value="1(2)"/>
 </reference>
 
+<reference anchor="mimesniff" target="https://mimesniff.spec.whatwg.org">
+   <front>
+      <title>MIME Sniffing</title>
+      <author>
+         <organization>WHAT Working Group</organization>
```

Comment:
Can this be WHATWG? Or do you also write IE Task Force?

Reply:
updated to WHATWG